components/eamxx/src/physics/nudging/atmosphere_nudging.cpp

```diff
@@ -212,26 +277,29 @@ void Nudging::run_impl (const int dt)
 
   //update time state information and check whether need to update data
   update_time_step(time_since_zero);
-  
+
   //perform time interpolation
   time_interpolation(time_since_zero);
 
   auto T_mid       = get_field_out("T_mid").get_view<mPack**>();
   auto qv          = get_field_out("qv").get_view<mPack**>();
-  auto hw          = get_field_out("horiz_winds").get_view<mPack***>();
+  auto u          = get_field_out("u").get_view<mPack**>();
+  auto v          = get_field_out("v").get_view<mPack**>();
 
   const auto& p_mid    = get_field_in("p_mid").get_view<const mPack**>();
   auto p_mid_v = view_Nd<mPack,2>("",p_mid.extent_int(0),p_mid.extent_int(1));
   Kokkos::deep_copy(p_mid_v,p_mid);
```

Comment:
I'm not sure I understand why you have this deep copy. Can't you do the same thing you do for the other views right below?

Reply:
Actually, there is a simpler way, just:
const auto& p_mid    = get_field_in("p_mid").get_view<const mPack**>();
And then p_mid_v can be just changed to p_mid everywhere